siphon/__init__.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2013-2015 University Corporation for Atmospheric Research/Unidata.
 # Distributed under the terms of the MIT License.
 # SPDX-License-Identifier: MIT
+"""Tools for accessing atmospheric and oceanic science data on remote servers."""
```

Comment:
Isn't AOS a bit limited considering lots of folks run THREDDS? But then again this isn't a THREDDS specific library. Maybe something like : Tools for accessing remote data hosted on THREDDS and other servers? Maybe that's too general?